### PR TITLE
docs: update documentation to clarify instance and management commands

### PR DESCRIPTION
**Pull request by wtrocki**
Number of  independent developers provided us feedback that documentation missing clarification what commands are used with instance and what are used for management. I have done this by adding documentation to top level cli objects.

Issue #901

### CODE REVIEW

File: pkg/cmd/registry/artifact/artifacts.go [hunk 1]
```diff
@@ -18,6 +18,8 @@ func NewArtifactsCommand(f *factory.Factory) *cobra.Command {
 		Use:   "artifact",
 		Short: "Manage Service Registry Artifacts",
 		Long: `
+Manage Service Registry Artifacts using currently selected Service Registry.
```
Comment (bhardesty) [inline pkg/cmd/registry/artifact/artifacts.go:21]:
```suggestion
Manage Service Registry Artifacts using the currently selected Service Registry.
```
If it's still technically accurate, consider replacing the "selected Service Registry" with "selected registry".

File: pkg/localize/locales/en/cmd/kafka.en.toml [hunk 2]
```diff
@@ -8,7 +8,14 @@ one = "Create, view, use, and manage your Kafka instances"
 
 [kafka.cmd.longDescription]
 description = "Long description for command"
-one = "A collection of commands for managing and interacting with Kafka instances."
+one = '''
+A collection of commands for managing and interacting with Kafka instances.
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka.en.toml:12]:
```suggestion
Commands for managing and interacting with Kafka instances.
```

File: pkg/localize/locales/en/cmd/kafka.en.toml [hunk 3]
```diff
@@ -8,7 +8,14 @@ one = "Create, view, use, and manage your Kafka instances"
...
+
+Commands are divided into the following categories:
+ - instance management commands: create, list etc.
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka.en.toml:15]:
```suggestion
 - instance management commands: create, list, and so on
```

File: pkg/localize/locales/en/cmd/kafka.en.toml [hunk 4]
```diff
@@ -8,7 +8,14 @@ one = "Create, view, use, and manage your Kafka instances"
...
+ - commands executed on selected instance: topic, consumer-group
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka.en.toml:16]:
```suggestion
 - commands executed on the selected instance: topic, consumer-group
```

File: pkg/localize/locales/en/cmd/kafka.en.toml [hunk 5]
```diff
@@ -8,7 +8,14 @@ one = "Create, view, use, and manage your Kafka instances"
...
+ - "use" command that selects current instance
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/kafka.en.toml:17]:
```suggestion
 - "use" command that selects the current instance
```

File: pkg/localize/locales/en/cmd/registry_crud.en.toml [hunk 6]
```diff
@@ -3,9 +3,14 @@ one = 'Service Registry commands'
 
 [registry.cmd.longDescription]
 one = ''' 
-Manage your Service Registry instances directly from the command line.
+Manage and interact with your Service Registry instances directly from the command line.
 
 Create new Service Registry instances and interact with them by adding schema and API artifacts and downloading them to your computer.
+
+Commands are divided into the following categories:
+ - instance management commands: create, list etc.
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/registry_crud.en.toml:11]:
```suggestion
 - instance management commands: create, list, and so on
```

File: pkg/localize/locales/en/cmd/registry_crud.en.toml [hunk 7]
```diff
@@ -3,9 +3,14 @@ one = 'Service Registry commands'
...
+ - commands executed on selected instance: artifacts
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/registry_crud.en.toml:12]:
```suggestion
 - commands executed on the selected instance: artifacts
```

File: pkg/localize/locales/en/cmd/registry_crud.en.toml [hunk 8]
```diff
@@ -3,9 +3,14 @@ one = 'Service Registry commands'
...
+ - "use" command that selects current instance
```
Comment (bhardesty) [inline pkg/localize/locales/en/cmd/registry_crud.en.toml:13]:
```suggestion
 - "use" command that selects the current instance
```

File: pkg/cmd/registry/artifact/artifacts.go (hunk 1 above)
Comment (wtrocki) [reply pkg/cmd/registry/artifact/artifacts.go:21]: There is huge chance of having another Registry in CLI so trying to be more specific. If we find any docs where we take shortcut it is best to change it

**Pull request merged**